docs/src/howto-guides/002_installation.md

```diff
@@ -131,6 +131,43 @@ Eg:
 dpkg -i tedge_mapper_0.5.0_armhf.deb
 ```
 
+## Uninstall `thin-edge.io`
+The `thin-edge.io` can be uninstalled using a script, the script can be downloaded
+from below mentioned location. 
+
+```shell
+https://raw.githubusercontent.com/thin-edge/thin-edge.io/main/uninstall-thin-edge_io.sh
```

Comment:
Shouldn't this be a `curl` or `wget` command?

Reply:
Yup, fixed it.